### ISSUE TITLE
Extra channel list not passed correctly to mulled-build

### ISSUE THREAD

**Issue by acaprez** [bioconda/bioconda-utils#31]
I noticed the `mulled-build` test was failing when testing a package that relied on dependencies from `conda-forge`.  Looking closer, the extra channel arguments in the `test_package` function of `pkg_test.py` are not formatted correctly.  The relevant snippet is:
```
for channel in channels:
    channel_args.extend(['--extra-channel', channel])
```
First, there's a typo there.  The argument name `mulled-build` accepts is `--extra-channels`; the trailing `s` is missing.  More important though, is that `mulled-build` expects a comma-separated list of channels and not multiple instances of the argument.  If I change the above to

`channel_args.extend(['--extra-channels', ','.join(channels)])`

then the channel list gets passed correctly and the test succeeds.

**Comment by daler** [comment]
Thanks @acaprez. I'll add this fix to https://github.com/bioconda/bioconda-utils/pull/30

@bgruening has it always been this syntax or has something recently changed?

**Comment by daler** [comment]
fixed in #30
